dmoj/executors/mixins.py

```diff
@@ -56,8 +96,8 @@ class PlatformExecutorMixin(metaclass=abc.ABCMeta):
     data_grace = 0
     fsize = 0
     personality = 0x0040000  # ADDR_NO_RANDOMIZE
-    fs: List[str] = []
-    write_fs: List[str] = []
+    fs: List[Rule] = []
```

Comment:
`Rule` is not descriptive here; it was fine inside `filesystem_policies.py`, but this is too far from that. Perhaps `Rule` should be renamed to something like `FilesystemAccessRule`.